### PR TITLE
created new module for start/stop ec2 instances by tags using schedul…

### PR DESCRIPTION
**Pull request by cristiansandru11**
…ed cloudwatch events

### CODE REVIEW

File: code/ec2_lambda_handler.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+start_instances_response = ec2.describe_instances(Filters=[
+    {
+        'Name': f'{start_tag_name}',
+        'Values': [
+            f'{start_tag_value}',
+        ]
+    },
+])
+
+# Getting instances tagged with Start tag
```
Comment (gteodoru) [inline code/ec2_lambda_handler.py]: this should be `Stop`

File: code/ec2_lambda_handler.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+
+def stop(event, context):
+    ec2.stop_instances(InstanceIds=stop_instances)
+    print('stopped instances: ' + str(stop_instances))
+
+
+def start(event, context):
+    ec2.start_instances(InstanceIds=start_instances)
+    print('started  instances: ' + str(start_instances))
```
Comment (gteodoru) [inline code/ec2_lambda_handler.py]: nothing should be global ... everything should be in methods 

```
def _get_instances_by_tag(tag_name, tag_value):
   .....

def _get_instance_ids(tag_name, tag_value):
  response = _get_instances_by_tag(tag_name, tag_value)
  .....

def stop(event, context):
  tag_name = os.getenv("STOP_TAG_NAME")
  tag_value = os.getenv("STOP_TAG_VALUE")
  instances = _get_instance_ids(tag_name, tag_name)
  ....

def start(event, context):
  tag_name = os.getenv("STOP_TAG_NAME")
  tag_value = os.getenv("STOP_TAG_VALUE")
  instances = _get_instance_ids(tag_name, tag_name)
  ....
```

Comment (gteodoru) [reply code/ec2_lambda_handler.py]: Add logger !!!!! for info

**Review by gteodoru — CHANGES_REQUESTED** [review]

File: code/ec2_lambda_handler.py (hunk 2 above)
Comment (cristiansandru11) [reply code/ec2_lambda_handler.py]: ❤️  10x

**Review by AndreiPencea — APPROVED** [review]

**Pull request merged**